src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -287,6 +287,64 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyRepoArtifacts"
+          AfterTargets="Package"
+          Condition="Exists($(RepoManifestFile))">
+
+    <XmlPeek XmlInputPath="$(RepoManifestFile)"
+             Query="Build/Artifact/@Path">
+        <Output TaskParameter="Result" ItemName="ArtifactPath" />
+    </XmlPeek>
+
+    <ItemGroup>
+      <_Asset Include="@(ArtifactPath)" Condition="'%(Extension)' != '.nupkg'"/>
+      <_Package Include="@(ArtifactPath)" Condition="'%(Extension)' == '.nupkg'"/>
+      <_NonShippingPackage Include="@(_Package)" Condition="$([System.Text.RegularExpressions.Regex]::Match(%(Identity),'[\\\\/]NonShipping[\\\\/]').Success)"/>
```

Comment:
I'm not sure we need a regex here. The directory separator char must be the same as on the current build system.

Consider doing something like `Contains('$([System.IO.Path]::DirectorySeparatorChar)NonShipping$([System.IO.Path]::DirectorySeparatorChar)')` here instead.

Reply:
Will fix.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18153/commits/9bfacadeccfd5235e7e99c2d624cb92b7807b176